packages/contracts/contracts/plugin/PluginRepo.sol

```diff
@@ -127,19 +130,19 @@ contract PluginRepo is
         bytes calldata _buildMetadata,
         bytes calldata _releaseMetadata
     ) external auth(address(this), MAINTAINER_PERMISSION_ID) {
-        // In a case where _pluginSetup doesn't contain supportsInterface,
-        // but contains fallback, that doesn't return anything(most cases)
-        // the below approach aims to still return custom error which not possible with try/catch..
-        // NOTE: also checks if _pluginSetup is a contract and reverts if not.
+        // If the passed address does not support [ERC-165](https://eips.ethereum.org/EIPS/eip-165)
+        // and has an empty fallback function, the following code will check if it is plugin setup
+        // and gracefully revert with an custom error if it is not.
         bytes memory data = _pluginSetup.functionCall(
             abi.encodeWithSelector(
                 ERC165Upgradeable.supportsInterface.selector,
                 type(IPluginSetup).interfaceId
             )
         );
 
-        // NOTE: if data contains 32 bytes that can't be decoded with uint256
-        // it reverts with solidity's ambigious error.
+        // TODO GIORGI FIXES THIS
```

Comment:
Here

Reply:
Yep, it's getting fixed in my PR. so I will remove this once I merge my PR. no worries.

Reply:
Closed as this will be fixed in another PR, persumably #229